specs/default/cluster-init/scripts/21-pmix.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 set -e
-PMIX_ROOT=/opt/pmix
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "$script_dir/../files/common.sh" 
+read_os
 
-logger -s "Configuring PMIx"
+PMIX_ROOT=/opt/pmix
 
+# TODO: Try not to change the slurmd configuration file, instead use the environment variables in the job script
 function configure_slurmd()
```

Comment:
Can we just remove this now? If not, can we put a # Deprecated comment on it, and explain in what situations someone may need it?